code/__DEFINES/jobs.dm

```diff
@@ -68,31 +68,32 @@
 #define JOB_DISPLAY_ORDER_QUARTERMASTER 4
 #define JOB_DISPLAY_ORDER_CARGO_TECHNICIAN 5
 #define JOB_DISPLAY_ORDER_SHAFT_MINER 6
-#define JOB_DISPLAY_ORDER_BARTENDER 7
-#define JOB_DISPLAY_ORDER_COOK 8
-#define JOB_DISPLAY_ORDER_BOTANIST 9
-#define JOB_DISPLAY_ORDER_JANITOR 10
-#define JOB_DISPLAY_ORDER_CLOWN 11
-#define JOB_DISPLAY_ORDER_MIME 12
-#define JOB_DISPLAY_ORDER_CURATOR 13
-#define JOB_DISPLAY_ORDER_LAWYER 14
-#define JOB_DISPLAY_ORDER_CHAPLAIN 15
-#define JOB_DISPLAY_ORDER_CHIEF_ENGINEER 16
-#define JOB_DISPLAY_ORDER_STATION_ENGINEER 17
-#define JOB_DISPLAY_ORDER_ATMOSPHERIC_TECHNICIAN 18
-#define JOB_DISPLAY_ORDER_CHIEF_MEDICAL_OFFICER 19
-#define JOB_DISPLAY_ORDER_MEDICAL_DOCTOR 20
-#define JOB_DISPLAY_ORDER_CHEMIST 21
-#define JOB_DISPLAY_ORDER_GENETICIST 22
-#define JOB_DISPLAY_ORDER_VIROLOGIST 23
-#define JOB_DISPLAY_ORDER_RESEARCH_DIRECTOR 24
-#define JOB_DISPLAY_ORDER_SCIENTIST 25
-#define JOB_DISPLAY_ORDER_ROBOTICIST 26
-#define JOB_DISPLAY_ORDER_HEAD_OF_SECURITY 27
-#define JOB_DISPLAY_ORDER_WARDEN 28
-#define JOB_DISPLAY_ORDER_DETECTIVE 29
-#define JOB_DISPLAY_ORDER_SECURITY_OFFICER 30
-#define JOB_DISPLAY_ORDER_BRIG_PHYS 31
-#define JOB_DISPLAY_ORDER_DEPUTY 32 
-#define JOB_DISPLAY_ORDER_AI 33
-#define JOB_DISPLAY_ORDER_CYBORG 34
+#define JOB_DISPLAY_ORDER_EXPLORER 7
+#define JOB_DISPLAY_ORDER_BARTENDER 8
+#define JOB_DISPLAY_ORDER_COOK 9
+#define JOB_DISPLAY_ORDER_BOTANIST 10
+#define JOB_DISPLAY_ORDER_JANITOR 11
+#define JOB_DISPLAY_ORDER_CLOWN 12
+#define JOB_DISPLAY_ORDER_MIME 13
+#define JOB_DISPLAY_ORDER_CURATOR 14
+#define JOB_DISPLAY_ORDER_LAWYER 15
+#define JOB_DISPLAY_ORDER_CHAPLAIN 16
+#define JOB_DISPLAY_ORDER_CHIEF_ENGINEER 17
+#define JOB_DISPLAY_ORDER_STATION_ENGINEER 18
+#define JOB_DISPLAY_ORDER_ATMOSPHERIC_TECHNICIAN 19
+#define JOB_DISPLAY_ORDER_CHIEF_MEDICAL_OFFICER 20
+#define JOB_DISPLAY_ORDER_MEDICAL_DOCTOR 21
+#define JOB_DISPLAY_ORDER_CHEMIST 22
+#define JOB_DISPLAY_ORDER_GENETICIST 23
+#define JOB_DISPLAY_ORDER_VIROLOGIST 24
+#define JOB_DISPLAY_ORDER_RESEARCH_DIRECTOR 25
+#define JOB_DISPLAY_ORDER_SCIENTIST 26
+#define JOB_DISPLAY_ORDER_ROBOTICIST 27
+#define JOB_DISPLAY_ORDER_HEAD_OF_SECURITY 28
+#define JOB_DISPLAY_ORDER_WARDEN 29
+#define JOB_DISPLAY_ORDER_DETECTIVE 30
+#define JOB_DISPLAY_ORDER_SECURITY_OFFICER 31
+#define JOB_DISPLAY_ORDER_BRIG_PHYS 32
+#define JOB_DISPLAY_ORDER_DEPUTY 33
+#define JOB_DISPLAY_ORDER_AI 34
+#define JOB_DISPLAY_ORDER_CYBORG 35

```

Comment:
protip, jak robisz takie coś to możesz na sam koniec dopisać i wtedy nie musisz przesuwać tego